docs/modules/wan/pages/modes.adoc

```diff
@@ -18,4 +18,6 @@ using active-active mode instead.
 * [[active-active]]**Active-Active:** Every cluster is equal, each cluster replicates to all other clusters.
 This is normally used to connect different clients to different clusters for the sake of
 the shortest path between client and server, hence gaining increased performance. An example use case would be geographically
-distributed applications.
+distributed applications. When using this mode of replication, if the same key is modified on multiple clusters, the data across
+all clusters will eventually match that of the last replicated transcation; this includes metadata such as expiry information
+if relevant.
```

Comment:
If the same key is modified on multiple clusters when using this mode of replication, data - including any relevant metadata such as expiry information - across all clusters eventually matches the last replicated transaction.

Reply:
Thanks, addressed in 04dbecf